core/src/test/java/org/elasticsearch/action/admin/indices/create/CreateIndexIT.java

```diff
@@ -367,8 +367,9 @@ public void testCreateShrinkIndexToN() {
     }
 
     public void testCreateShrinkIndex() {
-        internalCluster().ensureAtLeastNumDataNodes(2);
-        prepareCreate("source").setSettings(Settings.builder().put(indexSettings()).put("number_of_shards", randomIntBetween(2, 7))).get();
+        internalCluster().ensureAtLeastNumDataNodes(4);
+        prepareCreate("source").setSettings(Settings.builder().put(indexSettings())
```

Comment:
can we remove the `"number_of_replicas"` again since `indexSettings()` should take care of this